### ISSUE TITLE
Explicitly named members of an interface do not work with the `inheritdoc` tag

### ISSUE THREAD

**Issue by hegemonic** [jsdoc3/jsdoc#893]
In the following example, `FooClass` should inherit the documentation for `FooClass#doThing`. However, it does not:

```js
/**
 * @interface
 */
function FooInterface() {}

/**
 * Do a thing.
 * @method FooInterface#doThing
 */
FooInterface.prototype.doThing;

/**
 * @class
 * @implements {FooInterface}
 */
function FooClass() {}

/** @inheritdoc */
FooClass.prototype.doThing = function() {};
```

If you change `@method FooInterface#doThing` to `@method`, this example works fine. It appears that the explicit name is somehow causing JSDoc to get tripped up.

**Comment by hegemonic** [comment]
For the example shown above, we generate two doclets for `FooInterface#doThing`. One is for the virtual comment that includes `@method FooInterface#doThing`. The other is for the actual symbol in the code; that doclet has an `undocumented` property set to `true`.

The bug seems to be that `FooClass#doThing` is inheriting the docs from the undocumented version of `FooInterface#doThing` (or perhaps from both versions, but the last one wins).

**Comment by hegemonic** [comment]
Fixed on master and releases/3.3.
